docs/getting_started.md

```diff
@@ -0,0 +1,124 @@
+# Getting Started
+
+## System Prerequisites
+
+DAPper is written in Rust and requires the Rust toolchain. It is recommended to use the latest stable Rust version (1.70.0 or newer). You can install through your package manager or through [rustup.rs](https://www.rust-lang.org/tools/install).
```

Comment:
Not sure what our minimum requirement is for the rust toolchain -- it may be something like 1.85 or newer, since I think the other day I tried building on a fresh system with 1.81 and it didn't work until I updated to 1.88 (I didn't try versions in between). This seemed to be something to do with one of our dependencies not being happy with the older compiler.